test/metabase/upload_test.clj

```diff
@@ -1224,9 +1221,8 @@
                    (= :schema-filters (keyword (:type conn-prop))))
                  (driver/connection-properties driver))))
 
-(deftest create-csv-upload!-schema-does-not-sync-test
-  ;; Just test with postgres because failure should be independent of the driver
-  (mt/test-driver :postgres
+(deftest ^:mb/once create-csv-upload!-schema-does-not-sync-test
+  (mt/test-drivers (mt/normal-drivers-with-feature :uploads)
```

Comment:
I think this test implementation is pretty dependent on postgres specifics. Maybe this is best leaving

Reply:
Yeah it's failing with H2, and I wasn't sure why. I'll update the comment.